kinder/pkg/cluster/manager/actions/kubeadm-config.go

```diff
@@ -29,7 +29,8 @@ import (
 	"k8s.io/kubeadm/kinder/pkg/constants"
 	"k8s.io/kubeadm/kinder/pkg/cri"
 	"k8s.io/kubeadm/kinder/pkg/kubeadm"
-	kindkustomize "sigs.k8s.io/kind/pkg/kustomize"
+	kindkustomize "k8s.io/kubeadm/kinder/third_party/kind/util/kustomize"
```

Comment:
This should be `kindinternalkustomize` (we are using `kindinternal` prefix for everything we are forking)